newIDE/app/src/ProjectCreation/EmptyAndStartingPointProjects.js

```diff
@@ -0,0 +1,160 @@
+// @flow
+import * as React from 'react';
+import { I18n } from '@lingui/react';
+import { type StorageProvider, type SaveAsLocation } from '../ProjectsStorage';
+import Add from '../UI/CustomSvgIcons/Add';
+import Text from '../UI/Text';
+import { Trans } from '@lingui/macro';
+import { Column, Line, Spacer } from '../UI/Grid';
+import { type GDevelopTheme } from '../UI/Theme';
+import GDevelopThemeContext from '../UI/Theme/GDevelopThemeContext';
+import { ExampleTile } from '../AssetStore/ShopTiles';
+import { ExampleStoreContext } from '../AssetStore/ExampleStore/ExampleStoreContext';
+import { type ExampleShortHeader } from '../Utils/GDevelopServices/Example';
+import { useResponsiveWindowSize } from '../UI/Responsive/ResponsiveWindowMeasurer';
+import { GridList, GridListTile } from '@material-ui/core';
+import { shouldValidate } from '../UI/KeyboardShortcuts/InteractionKeys';
+import classes from './EmptyAndStartingPointProjects.module.css';
+import classNames from 'classnames';
+import { getItemsColumns } from './NewProjectSetupDialog';
+
+const getStyles = (theme: GDevelopTheme) => ({
+  grid: {
+    margin: 0,
+    // Remove the scroll capability of the grid, the scroll view handles it.
+    overflow: 'unset',
+  },
+  cellSpacing: 2,
+});
+
+type EmptyProjectTileProps = {|
+  onSelectEmptyProject: () => void,
+  disabled?: boolean,
+  /** Props needed so that GridList component can adjust tile size */
+  style?: any,
+|};
+
+// The design of this tile copies the ones in ShopTiles.js
+const EmptyProjectTile = ({
+  onSelectEmptyProject,
+  disabled,
+  style,
+}: EmptyProjectTileProps) => {
+  const { isMobile } = useResponsiveWindowSize();
+  return (
+    <GridListTile style={style}>
+      <div
+        className={classNames({
+          [classes.container]: true,
+        })}
+      >
+        <div
+          className={classNames({
+            [classes.emptyProject]: true,
+          })}
+          onClick={disabled ? undefined : onSelectEmptyProject}
+          tabIndex={0}
+          onKeyPress={(event: SyntheticKeyboardEvent<HTMLLIElement>): void => {
+            if (shouldValidate(event) && !disabled) {
+              onSelectEmptyProject();
+            }
+          }}
+        >
+          <Column alignItems="center" justifyContent="center" expand>
+            <Add />
+            <Text align="center" noMargin>
+              <Trans>Empty project</Trans>
+            </Text>
+          </Column>
+        </div>
+        <Column>
+          {isMobile && <Spacer />}
+          <Line justifyContent="flex-start" noMargin>
+            {/* Add a hidden text to match the height of the other tiles on the row */}
+            <Text size="body2" hidden noMargin={isMobile}>
+              <Trans>Empty project</Trans>
+            </Text>
+          </Line>
+        </Column>
+      </div>
+    </GridListTile>
+  );
+};
+
+export const getStartingPointExampleShortHeaderTitle = (
```

Comment:
Do you think we would need a user friendly name field?

Reply:
I'm not sure. Maybe. The problem here is that we already have an example called "Platformer" and we want to display platformer, but the title is unique in our repo.
So maybe we could use a user-friendly title indeed.
Though, I expect all "starting-points" to start the same way so we can manage like this